### ISSUE TITLE
Remove SelectField element and styles

### ISSUE THREAD

**Issue by annatuma** [cc-archive/vocabulary-legacy#70]
Blocked by: https://github.com/creativecommons/vue-vocabulary/issues/117

We are moving one SelectField style up to a new "Form" element. The rest we should remove. This should not happen until issue #117 is complete.

**Comment by kgodey** [comment]
No longer relevant with new repo changes.
